Bottom: b2d529be1a7b43c622d3c288dc0de67df0bfb421
Top:    df87cc5c7173bc15b1aad2a8d84e5528bf412636
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-02-17 19:44:25 +0800

Refresh of virtio-net-small-buffer-xdp

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index 765b65e..5fd6319 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -1979,16 +1979,6 @@ static void free_receive_page_frags(struct virtnet_info *vi)
 			put_page(vi->rq[i].alloc_frag.page);
 }
 
-static bool is_xdp_raw_buffer_queue(struct virtnet_info *vi, int q)
-{
-	if (q < (vi->curr_queue_pairs - vi->xdp_queue_pairs))
-		return false;
-	else if (q < vi->curr_queue_pairs)
-		return true;
-	else
-		return false;
-}
-
 static void free_unused_bufs(struct virtnet_info *vi)
 {
 	void *buf;
